### DISCUSSION TITLE
Hashmaps support

### DISCUSSION THREAD

**Discussion by DypoB** [react-hook-form/react-hook-form#6366] [Ideas]
Hi, I want to suggest a syntax for field names where i want to use Hashmap Object with ids

Like that: { 1: true, 21999: false }

roles.map(role => (
   <CheckboxField name="roleIds.{${role.id}}" />
))

Because in the same situations with Array, we should use [] instead of {}

**Comment by DypoB** [comment]
something get wrong with example:

 `<CheckboxField name = "roleIds.{${Role.id}}" />`

**Reply by bluebill1049** [reply]
did you resolve your issue?
